app/controllers/adventures_controller.rb

```diff
@@ -45,6 +44,10 @@ def create
     if @adventure.save
       flash[:notice] = "Excellent! Another adventure awaits!"
       redirect_to bucket_list_path(@bucket_list)
+    elsif @adventure.id == nil && @bucket_list.id != nil
```

Comment:
Prefer the use of the `nil?` predicate.<br>Prefer `!expression.nil?` over `expression != nil`.